src/components/Table.tsx

```diff
@@ -61,7 +62,8 @@ const BaseTable = styled.div`
 				text-align: left;
 				padding-top: 14px;
 				padding-bottom: 14px;
-				padding-left: 40px;
+				padding-left: 20px;
+				padding-right: 20px;
```

Comment:
NIT: since we are now defining all padding sides, how about using the shorthand `padding: 14px 20px;` (and the respective values in the above diff chunk)?